README.md

```diff
@@ -27,6 +27,20 @@ documentation at https://articlequality.readthedocs.io .
                      'ga': 0.21617801281707663,
                      'start': 0.13906813268582238}}
 
+## Install
+
+### Requirements
+
+* Python 3.5, 3.6 or 3.7
```

Comment:
We also support python 3.8.

Reply:
I'm not sure that this is robustly true. Support for Python 3.8 was added in [SciPy 1.3.2](https://github.com/scipy/scipy/releases/tag/v1.3.2) (see also scipy/scipy#10927) and revscoring's SciPy requirement is `scipy >= 1.2.3, < 1.2.999`. In practice, that means that some Fortran compilation step in SciPy 1.2 fails with Python 3.8, but succeeds with Python 3.7.

Reply:
Interesting!  Our test suite all passes with python 3.8.  It's part of our CI pipeline.